app/dashboard/static/js/app/view-tests-case-id.2020.3.js

```diff
@@ -0,0 +1,407 @@
+            modulesNode.appendChild(document.createTextNode(results.modules));
+            modulesNode.href =
+                translatedURI[0].path(modulesPath).normalizePath().href();
+            modulesNode.insertAdjacentHTML('beforeend', '&nbsp;');
+            modulesNode.appendChild(html.external());
+        }
+
+        html.replaceContent(
+            document.getElementById('defconfig'), defconfigNode);
+        html.replaceContent(
+            document.getElementById('endian'),
+            document.createTextNode(results.endian));
+        html.replaceContent(
+            document.getElementById('git-url'), gitNode);
+        html.replaceContent(
+            document.getElementById('compiler'),
+            document.createTextNode(results.compiler_version_full));
+        html.replaceContent(
+            document.getElementById('job-log'), logNode);
+        html.replaceContent(
+            document.getElementById('kernel-image'), kernelNode);
+        html.replaceContent(
+            document.getElementById('modules'), modulesNode);
+        html.replaceContent(
+            document.getElementById('dtb'), dtbNode);
+    }
+
+    function addRegressionHistory(results) {
+        var data;
+        var columns;
+
+        data = [];
+        results.regressions.reverse().forEach(function(item) {
+            data.push({
+                'kernel': item.kernel,
+                'created_on': item.created_on,
+                'status': item.status,
+                'test_case_id': item.test_case_id.$oid,
+            });
+        });
+
+        function _renderStatus(data, type) {
+            if (type == "display") {
+                return ttable.statusNode(data).outerHTML;
+            } else {
+                return data;
+            }
+        }
+
+        /* ToDo: highlight current revision row, and show "future" regressions
+         * when looking at other revisions (needs database schema changes).
+         */
+        columns = [
+            {
+                data: 'kernel',
+                title: 'Kernel',
+                type: 'string',
+                className: 'kernel-column',
+            },
+            {
+                data: 'created_on',
+                title: 'Date',
+                type: 'date',
+                orderable: false,
+                className: 'date-column-count pull-center',
+                render: ttable.renderDate,
+            },
+            {
+                data: 'status',
+                title: 'Status',
+                type: 'string',
+                searchable: false,
+                orderable: false,
+                className: 'pull-center',
+                render: _renderStatus,
+            },
+        ]
+
+        gRegressionTable
+            .data(data)
+            .columns(columns)
+            .languageLengthMenu('Revisions per page')
+            .lengthMenu([10, 25, 50, 75, 100])
+            .rowURL('/test/case/id/%(test_case_id)s')
```

Comment:
it still a little bit weird to me a link that just "reload" the page, but it is not a big problem. I see.

Reply:
It would be more confusing to not have the current revision in the table.

Reply:
Maybe we can have a view to show something that could be interesting for kernel developers related to a test case in the next round of improvements. I am just thinking, I don't know what kind of thing could be useful.. :}